torch/onnx/symbolic_opset9.py

```diff
@@ -471,8 +471,17 @@ def expand_as(g, self, other):
     return g.op("Expand", self, shape)
 
 
+@parse_args('v', 'v', 'i', 'v', 'v')
 def embedding(g, weight, indices, padding_idx, scale_grad_by_freq, sparse):
-    return g.op("Gather", weight, indices)
+    # if (padding_idx >= 0) {
```

Comment:
What does this mean? Seems that for `padding_idx >= 0`, the pytorch statement becomes `embedding.masked_fill_((indices == padding_idx).reshape({-1, 1}), 0);` for embedding operator. If this is the case, maybe it's better to put it in the `if (padding_idx >= 0):` block below.

Reply:
I guess this is reference code for pytorch implementation. Might be better if this is clarified in comments.